### ISSUE TITLE
AST JSON: Constants are not present

### ISSUE THREAD

**Issue by Caleb-o** [c3lang/c3c#1540]
Constants seem to not be emitted into the JSON at all, when they should be.

**Comment by Cardisk** [comment]
Hi, new here. I was trying to solve this issue and I also added the type to the globals as well. I will try to add also the init value. Is it ok if I collapse the two issues in only one?
By the way this is my first contribution ever, there are any instructions on how to contribute to this project beside the code of conduct and the codestyle files?

**Comment by lerno** [comment]
Not really, just rebase the commit so that it's a single commit. And add a line in the release notes for the change.

**Comment by lerno** [comment]
Is this fixed?

**Comment by cardisk** [comment]
Yes. There are a few expressions that I really cannot trigger so I wasn't able to output a specific value for them, instead I outputted a `TODO:` with the corresponding type so in the future it can be easy to implement them. At the moment there are no TODOs inside the standard library.
If this issue can be closed I think also the #1541 one should, because inside the AST constants and globals have the same output.

**Comment by lerno** [comment]
Also closing this, as we're reworking what it should contain, see #2349
